id.go

```diff
@@ -0,0 +1,15 @@
+package imap
+
+type IdData struct {
```

Comment:
Nit: can we use the proper upper-casing for things like "ID", "OS", "URL"?

See https://go.dev/wiki/CodeReviewComments#initialisms

(The "ID" thing applies to all symbols: `IDData`, `IDCommand`, `Client.ID`, and so on)